content_restrictions/static/html/children.html

```diff
@@ -1,3 +1,7 @@
 <div class="content_restrictions_block">
-    {% for child_content in children_contents %} {{ child_content|safe }} {% endfor %}
+    {% for child_content in children_contents %}
+        {{ child_content|safe }}
+        <br>
+    {% endfor %}
+    <br>
```

Comment:
I think we could add a separator between each component

```suggestion
    {% for child_content in children_contents %}
        {{ child_content|safe }}
        <br>
        <hr>
    {% endfor %}
```

Reply:
Yes! That looks even better. Thank you!